daml/Marketplace/Distribution/Auction/Service.daml

```diff
@@ -157,11 +157,7 @@ template CreateAuctionRequest
     operator : Party
     provider : Party
     customer : Party
-    auctionId : Text
-    asset : Asset
-    quotedAssetId : Id
-    floorPrice : Decimal
-    assetDepositCid : ContractId AssetDeposit
+    request : RequestCreateAuction
```

Comment:
@lucianojoublanc-da - Should we add back the individual fields so there's no need to extract and then create the fields when they are being used to create the `Auction` ?

Reply:
Yeah, so following on from the above comments, in this particular case, removing the `..` didn't help too much because the `Auction` type has the extra `status` field which is external. So in this case removing the `..` hasn't helped too much. I would probably leave it as it originally was, or possibly expand the `{..}` with the field names, and then do `Auction with auctionId, asset, quotedAssetId` etc. I don't know. I think this case is not as clear cut as others where we're copying every single field ...

Reply:
>  I would probably leave it as it originally was, or possibly expand the {..} with the field names, and then do `Auction with auctionId, asset, quotedAssetId`
 
 This is the approach we used in spider, if I recall correctly.